Shared/Samples/Group layers together/GroupLayersTogetherView.swift

```diff
@@ -54,27 +54,18 @@ struct GroupLayersTogetherView: View {
             }
             .toolbar {
                 ToolbarItem(placement: .bottomBar) {
-                    layersButton
+                    Button("Layers") {
+                        isShowingLayersSheet = true
```

Comment:
Yea I didn't updating the naming at all in PR since I personally use sheet and popover interchangeably (mostly because they present the same on iOS). I definitely can update these here or in a separate PR if you think it is important.

Reply:
Good to create a separate issue for update naming. I would suggest using appropriate function name in variable or type names rather than `sheet`. For example, you can name this as `isShowingLayersList` or `isShowingLayersView`.

Reply:
Issue logged in `swift/issues/5234`.